src/main/java/vazkii/botania/common/core/handler/SleepingHandler.java

```diff
@@ -11,18 +11,20 @@
 
 @Mod.EventBusSubscriber(modid = LibMisc.MOD_ID)
 public final class SleepingHandler {
-	
+
 	private SleepingHandler() {}
 
 	@SubscribeEvent
 	public static void trySleep(PlayerSleepInBedEvent event) {
-		World world = event.getPlayer().world;
-		boolean nearGuardian = ((ServerWorld) world).getEntities()
-				.filter(e -> e instanceof EntityDoppleganger)
-				.anyMatch(e -> ((EntityDoppleganger) e).getPlayersAround().contains(event.getPlayer()));
+    World world = event.getPlayer().world;
+    if(!world.isRemote()) {
+      boolean nearGuardian = ((ServerWorld) world).getEntities()
+          .filter(e -> e instanceof EntityDoppleganger)
+          .anyMatch(e -> ((EntityDoppleganger) e).getPlayersAround().contains(event.getPlayer()));
 
-		if(nearGuardian) {
-			event.setResult(PlayerEntity.SleepResult.NOT_SAFE);
-		}
+      if(nearGuardian) {
+        event.setResult(PlayerEntity.SleepResult.NOT_SAFE);
+      }
+    }
```

Comment:
Don't change formatting to spaces

Reply:
Hmm my IDE normally adapts to indent in the file ... thanks for the hint.